spec/unit/virtus/attribute/collection/value_coerced_predicate_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+require 'set'
+
+describe Virtus::Attribute::Collection, '#value_coerced?' do
+  subject { object.value_coerced?(input) }
+
+  let(:object) { described_class.build(Array[Integer]) }
+
+  context 'when input has correctly typed members' do
+    let(:input) { [1,2,3] }
```

Comment:
How can you enforce rubocop? Do you mean connecting it to https://houndci.com/ ?